### PR TITLE
minor fixes, styling, and logic bugs

### PR DESCRIPTION
**Pull request by Veivel**
## Summary 

Completed as many stories as I can from the bottom of backlog as well as fix some other issues I encountered along the way.

## Changes

- Pin docker image version of victoriametrics in `infra/stg/docker-compose.yml`
- Fix login issue, where google redirected user to `/auth/api/google/callback` but the page was blocked by auth middleware
- Fix navbar in Create Document page
- Add `is_admin` data to context for several templates in `bot` and `document` module
- Remove data source field from create chatbot and edit chatbot page, as well as from JS files used
- Fix issue where after being redirected by google login, user email was saved regardless if it was valid broom.id email or not
- Add proper "error page" for non-broom.id emails logging in

### CODE REVIEW

File: infra/stg/docker-compose.yml [hunk 1]
```diff
@@ -72,7 +72,7 @@ services:
       - public-web
 
   sapungobrol-victoriametrics:
-    image: victoriametrics/victoria-metrics:latest
+    image: victoriametrics/victoria-metrics:v1.106.0
```
Comment (Insta-x) [inline infra/stg/docker-compose.yml:75]: any reason to pin the version? or it's just to make sure that it won't break in the future?

Comment (Veivel) [reply infra/stg/docker-compose.yml:75]: Yep, you got it. Having the `latest` tag was never the intent, I left that out by accident.

**Review by TakeMeGH — APPROVED** [review]

**Review by jasonirvine76 — APPROVED** [review]
Seems good to me, i dont have any issues with the code

**Comment by nyoosteven** [comment]
Seems good to me, i dont have any issues with the code

**Pull request merged**